<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>812</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="checkBoxHighlighting">
      <property name="text">
       <string>Highlighting</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="lineWrapMode">
       <enum>QTextEdit::WidgetWidth</enum>
      </property>
      <property name="lineWrapColumnOrWidth">
       <number>0</number>
      </property>
      <property name="tabStopWidth">
       <number>40</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>875</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCreate">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionCreateFile"/>
    <addaction name="actionOpenFile"/>
    <addaction name="actionSaveFile"/>
   </widget>
   <widget class="QMenu" name="menuFont">
    <property name="title">
     <string>Text</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionFontSetting"/>
    <addaction name="actionChangeFontColor"/>
    <addaction name="actionChangeBackgroundColor"/>
    <addaction name="actionVidelenieColor"/>
    <addaction name="separator"/>
    <addaction name="actionSetCodec"/>
    <addaction name="actionSetTabStop"/>
   </widget>
   <widget class="QMenu" name="menuCompile">
    <property name="title">
     <string>Compile</string>
    </property>
    <addaction name="actionCompileGpp"/>
   </widget>
   <addaction name="menuCreate"/>
   <addaction name="menuFont"/>
   <addaction name="menuCompile"/>
  </widget>
  <action name="actionCreateFile">
   <property name="text">
    <string>Create file</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="text">
    <string>Save file</string>
   </property>
  </action>
  <action name="actionChangeFontColor">
   <property name="text">
    <string>Change font color</string>
   </property>
  </action>
  <action name="actionChangeFontSize">
   <property name="text">
    <string>Change font size</string>
   </property>
  </action>
  <action name="actionChangeBackgroundColor">
   <property name="text">
    <string>Change background color</string>
   </property>
  </action>
  <action name="actiongpp">
   <property name="text">
    <string>g++</string>
   </property>
  </action>
  <action name="actionSetTabStop">
   <property name="text">
    <string>Set tab stop</string>
   </property>
  </action>
  <action name="actionHighlite">
   <property name="text">
    <string>Highlite</string>
   </property>
  </action>
  <action name="actionFontSetting">
   <property name="text">
    <string>Font setting</string>
   </property>
  </action>
  <action name="actionCompileGpp">
   <property name="text">
    <string>Compile gpp</string>
   </property>
  </action>
  <action name="actionVidelenieColor">
   <property name="text">
    <string>Background color</string>
   </property>
  </action>
  <action name="actionSetCodec">
   <property name="text">
    <string>Set codec</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
